### ISSUE TITLE
Kein Inhalt nach Überschrift wegen rechter Spalte

### ISSUE THREAD

**Issue by AndiWebred** [RRZE-Webteam/FAU-Einrichtungen#1383]
Siteimprove gibt mir für Seiten mit rechter Spalte einen Fehler aus: "Nach der Überschrift fehlt Inhalt". Anscheinend weil nach der normalen H2 gleich die H2 der rechten Spalte "gelesen" wird.

Beispiel: https://www.fau.de/education/bewerbung/ 
Siteimprove-Fehler-Seite: https://my2.siteimprove.com/Inspector/705695/A11Y/Page?pageId=60788627380&impmd=CDF7C5F66B15D04FC961306C8B003AD3&conf=a+aa+aaa+si&decision=none#/sia-r78/failed

**Comment by lukasniebler** [comment]
Grund:
Das Feld Subtitle der Website wird über der Sidebar ausgegeben. Wenn in der Sidebar jetzt ein Titel für das Textfeld oben angelegt wird folgen 2 H2-Elemente aufeinander.

**Comment by xwolfde** [comment]
in dev
